Tools/MonoGame.Content.Builder/Program.cs

```diff
@@ -45,6 +47,17 @@ static int Main(string[] args)
                     Console.Error.WriteLine("The debugger is not implemented under Mono and thus is not supported on your platform.");
                 }
             }
+            if (content.WaitForDebuggerToAttach)
+            {
+                var currentProcess = Process.GetCurrentProcess();
+                Console.WriteLine($"Waiting for debugger to attach ({currentProcess.MainModule.FileName} PID {currentProcess.Id}).  Press any key to continue without debugger.");
+                while (!Debugger.IsAttached) {
```

Comment:
Braces should be be in new lines.

Reply:
Agreed, let us use the same braces formatting as the rest of MonoGame, braces on new lines, indented with the function.

Other than that, it is good to go.